core/arch/arm/mm/tee_mmu.c

```diff
@@ -694,12 +715,19 @@ TEE_Result vm_unmap(struct user_mode_ctx *uctx, vaddr_t va, size_t len)
 	struct vm_region *r = NULL;
 	struct vm_region *r_next = NULL;
 	size_t l = ROUNDUP(len, SMALL_PAGE_SIZE);
```

Comment:
No point in initializing this here (other than setting to 0 for example) anymore after your changes, right?

Reply:
Right, I'll fix.